packages/babel-preset-env/test/targets-parser.spec.js

```diff
@@ -84,6 +94,41 @@ describe("getTargets", () => {
         safari: "tp",
       });
     });
+
+    it("works with android", () => {
+      expect(
+        getTargets({
+          browsers: "Android 4",
+        }),
+      ).toEqual({
+        android: "4.0.0",
+      });
+    });
+
+    it("works with inequalities", () => {
+      expect(
+        getTargets({
+          browsers: "Android >= 4",
+        }),
+      ).toEqual({
+        android: "4.0.0",
+      });
+    });
+
+    it("ignores invalid", () => {
+      expect(
+        getTargets({
+          browsers: 59,
```

Comment:
I merged this and it's failing on this?

Reply:
I will fix it.

Reply:
@hzoo The test was removed in previous commits (https://github.com/babel/babel/commit/3de053cc6c94be9411deb213cfc53c1392b5f6eb) and was changed to "throws on invalid browsers target". I will prepare PR that will remove this test.

Reply:
https://github.com/babel/babel/pull/8149